src/IB/IBFESurfaceMethod.cpp

```diff
@@ -513,6 +717,7 @@ IBFESurfaceMethod::interpolateVelocity(const int u_data_idx,
         NumericVector<double>* U_t_vec = NULL;
         NumericVector<double>* X_vec = NULL;
         NumericVector<double>* X_ghost_vec = d_X_IB_ghost_vecs[part];
+        boost::array<PetscVector<double>*, NDIM> DU_j_ghost_vec = d_DU_j_IB_ghost_vecs[part];
```

Comment:
Here and elsewhere please use `std::array`: its the same thing, but now that we have C++11 we can use the 'real' class.